community/stats/index.md

```diff
@@ -33,7 +33,7 @@ Spinnaker's telemetry module collects the following bits of data:
   * Each stage's:
     * Final status
     * Type
-    * Cloud Provider (if whitelisted)
+    * Cloud Provider
```

Comment:
I just removed this particular instance, as it didn't seem like it was providing useful/helpful information.

In all of these cases (status, type, cloud provider) we only send along the field if we recognize it (ie, if it's a cloud provider we've heard of or a stage type we know about) and send "UNKNOWN" otherwise.  Maybe there's a way of saying that in a separate paragraph that makes it clear, but as it was I didn't really understand what that meant without looking at the code (and it applies to more than just the cloud provider).

Reply:
Good point that this applies to all these items, not just provider. 
It [_looks like_](https://github.com/spinnaker/spinnaker.github.io/pull/1640) this is a setting that can be enabled/disabled (enabled by default, as of 1.19).

Reply:
Yes, stats collection is indeed controllable by a setting (which is now enabled by default).

The issue here is whether, if someone built a custom cloud provider named `my-sensitive-cloud-provider`, if we would send that cloud provider name as part of the stats collection.   (Similarly if I made a custom stage called `eric-custom-stage-secret`, would we send that along.)

The answer in both cases is that we won't, because we have a list of known providers/stages and the client code only sends along those we know about, and just sends along `UNKNOWN` if it's not one in that list.

Let me know if you think we should add something to this effect in that section; I'm not sure if this is helpful information or just going to confuse people.